src/allotropy/allotrope/schema_mappers/adm/liquid_chromatography/benchling/_2023/_09/liquid_chromatography.py

```diff
@@ -0,0 +1,389 @@
+    ChromatogramDataCube,
+    ChromatographyColumnDocument,
+    DerivedColumnPressureDataCube,
+    DeviceControlAggregateDocument,
+    DeviceControlDocumentItem,
+    DeviceSystemDocument,
+    InjectionDocument,
+    LiquidChromatographyAggregateDocument,
+    LiquidChromatographyDocumentItem,
+    MeasurementAggregateDocument,
+    MeasurementDocument,
+    Model,
+    PostColumnPressureDataCube,
+    PreColumnPressureDataCube,
+    ProcessedDataAggregateDocument,
+    ProcessedDataDocumentItem,
+    SampleDocument,
+    SampleFlowRateDataCube,
+    SamplePressureDataCube,
+    SolventConcentrationDataCube,
+    SystemFlowRateDataCube,
+    SystemPressureDataCube,
+    TemperatureProfileDataCube,
+)
+from allotropy.allotrope.models.shared.definitions.custom import (
+    TQuantityValueCubicMillimeter,
+    TQuantityValueMicrometer,
+    TQuantityValueMillimeter,
+)
+from allotropy.allotrope.models.shared.definitions.definitions import (
+    FieldComponentDatatype,
+    TDatacube,
+    TDatacubeComponent,
+    TDatacubeData,
+    TDatacubeStructure,
+)
+from allotropy.allotrope.schema_mappers.schema_mapper import SchemaMapper
+
+
+@dataclass(frozen=True)
+class Metadata:
+    asset_management_id: str
+    product_manufacturer: str
+    device_id: str
+    firmware_version: str
+    analyst: str
+
+
+@dataclass(frozen=True)
+class DataCubeComponent:
+    type_: FieldComponentDatatype
+    concept: str
+    unit: str
+
+
+@dataclass(frozen=True)
+class DataCube:
+    label: str
+    structure_dimensions: list[DataCubeComponent]
+    structure_measures: list[DataCubeComponent]
+    dimensions: list[tuple[float, ...]]
+    measures: list[tuple[float | None, ...]]
+
+
+@dataclass(frozen=True)
+class ProcessedDataDoc:
+    chromatogram_data_cube: DataCube | None = None
+    derived_column_pressure_data_cube: DataCube | None = None
+
+
+@dataclass(frozen=True)
+class DeviceControlDoc:
+    device_type: str
+    solvent_conc_data_cube: DataCube | None = None
+    pre_column_pressure_data_cube: DataCube | None = None
+    sample_pressure_data_cube: DataCube | None = None
+    system_pressure_data_cube: DataCube | None = None
+    post_column_pressure_data_cube: DataCube | None = None
+    sample_flow_data_cube: DataCube | None = None
+    system_flow_data_cube: DataCube | None = None
+    temperature_profile_data_cube: DataCube | None = None
+
+
+@dataclass(frozen=True)
+class ChromatographyDoc:
+    chromatography_serial_num: str
+    column_inner_diameter: float
+    chromatography_chemistry_type: str
+    chromatography_particle_size: float
+
+
+@dataclass(frozen=True)
+class InjectionDoc:
+    injection_identifier: str
+    injection_time: str
+    autosampler_injection_volume_setting: float
+
+
+@dataclass(frozen=True)
+class SampleDoc:
+    sample_identifier: str
+    batch_identifier: str
+
+
+@dataclass(frozen=True)
+class Measurement:
+    measurement_identifier: str
+    chromatography_column_doc: ChromatographyDoc
```

Comment:
For schema mapper `Measurement` classes, we generally want the structure to be as flat as possible. 

General rule of thumb is, if the item is a list (or had additional structure, like data cubes) there should be a new dataclass, otherwise, just combine here.

So fields from `ChromatographyDoc`, `SampleDoc`, and `InjectionDoc` should just be merged into `Measurement` here.

I know you are using these dataclasses in some spots in the parser and that will have to be re-tooled a bit, but we want to keep this pattern consistent across schema mappers and independent of parser implementation.

For the parser implementation, I would recommend having the class that nests these sub-docs just store the fields directly.